MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"bazel-contrib/rules_oci"
+
+module(
+    name = "contrib_rules_oci",
+    compatibility_level = 1,
+    version = "0.0.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "aspect_bazel_lib", version = "1.24.2")
+# NOTE: fails because rules_pkg has not been publishing to BCR
+bazel_dep(name = "rules_pkg", version = "0.7.1")
```

Comment:
Version 0.8.1 is available as of 8 February 2023, per your merge of bazelbuild/bazel-central-registry#425.